security_privacy_ln.asciidoc

```diff
@@ -0,0 +1,614 @@
+However, the LN operates under a different security model, which introduces novel security and privacy challenges.
+
+= Definitions of privacy =
+
+Privacy is a complex topic.
+It is hard to precisely define what we mean by privacy.
+Therefore, the question: "Is Lightning private?" has no direct answer.
+Privacy researchers approach this issue as follows.
+First, they define a _security model_ that specifies what an adversary is capable of and aims to achieve.
+Then, they describe the relevant properties of the system and check whether it conforms to the requirements.
+A security model is based on a set of underlying _security assumptions_.
+In cryptographic systems, they are often centered around the mathematical properties of the cryptographic primitives such as ciphers, signatures, and hash functions.
+The security assumptions of the Lightning Network are that the ECDSA signatures, SHA-256 hash function, and other cryptographic functions used in the protocol behave within their security definitions.
+For example, we assume that it is practically impossible to find a pre-image of a hash function.
+This allows the LN to rely on the HTLC mechanism for the atomicity of multi-hop payments: nobody except the final recipient can reveal the payment secret and resolve the HTLC.
+We also assume a degree of connectivity in the network, namely, that Lightning channels form a connected graph (therefore, it is possible to find a path from any sender to any receiver) and that network messages get propagated within certain timeouts.
+
+Here are some possible models of adversaries in the Lightning Network.
+A regular "honest-but-curious" forwarding node can observe payment amounts, the immediately preceding and following nodes, the graph of announced channels with their capacities.
+A very well-connected node can do the same but to a larger extent.
+For example, consider the developers of a popular wallet who maintain a node that their users connect to by default.
+This node would be responsible for routing a large share of payments to and from the users of that wallet.
+An even stronger security model implies that multiple nodes are under adversarial control.
+If it so happens that two colluding nodes are on the same payment path, they would understand that they are forwarding HTLCs belonging to the same payment, as such HTLCs have the same payment hash.
+
+What may be the goals of a Lightning attacker?
+Information security is often described in terms of three main properties: confidentiality, integrity, and availability.
+Confidentiality means that the information only gets to intended recipients.
+Integrity implies that the information does not get altered in transit.
+Availability ensures that users may expect the system to be functioning most of the time.
+
+The important properties of the Lightning Network are mostly centered around confidentiality and availability.
+Here are some of the properties we may want to ensure:
```

Comment:
Who's "we" here, it reads as if the authors are attempting to retroactively apply their own security models to LN, rather than attempt to analyze the details of the system and the resulting emerging security attributes of it.